src/components/Tour/Tour.vue

```diff
@@ -131,6 +131,10 @@ export default {
       localStorage.setItem('corteza.tour', JSON.stringify(false))
       this.$tours[this.tour.name].stop()
     },
+    onStartClick () {
```

Comment:
Can we please rename the start method to onStart, and use that one for this?
I don't understand why we would use a different method